### ISSUE TITLE
libgomp on musl: "Error relocating /lib/libgomp.so.1: __cxa_finalize: initial-exec TLS resolves to dynamic definition in /lib/libgomp.so.1"

### ISSUE THREAD

**Issue by Gottox** [void-linux/void-packages#3924]
### System

* xuname:  
  Void 4.18.15_1 x86_64-musl GenuineIntel uptodate DFF
* package:  
  * **libgomp**
  * ImageMagick (https://github.com/void-linux/void-packages/pull/3696)
  * libimagequant
  * python3-Pillow

### Expected behavior
packages linking to libgomp should work properly.

### Actual behavior
Packages linking against libgomp are prone to error out with
```"Error relocating /lib/libgomp.so.1: __cxa_finalize: initial-exec TLS resolves to dynamic definition in /lib/libgomp.so.1"```. 

### Steps to reproduce the behavior
```
$ python3 
Python 3.6.6 (default, Jul 16 2018, 08:54:52) 
[GCC 7.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from PIL import Image
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.6/site-packages/PIL/Image.py", line 64, in <module>
    from . import _imaging as core
ImportError: Error relocating /lib/libgomp.so.1: __cxa_finalize: initial-exec TLS resolves to dynamic definition in /lib/libgomp.so.1
>>> 
```

**Comment by OliverEvans96** [comment]
Has this been resolved? Any advice? Thanks!

**Comment by ericonr** [comment]
@OliverEvans96 if this is happening for you in Void, open a new issue. It has been resolved with the patch in the commit above.

> closed this in 75fe6cb
